Expose getArgumentValues from root

`getArgumentValues` is marked as internal and only exposed when you import from `graphql/execution/values`.

If you could expose it from the root `index.js` it would make importing it far easier in contexts without imports to relative files (a cdn for my case.)

The function is already used in production ready code in joinmonster and is necessary to get arguments from resolve info in field handlers. I don't see a good reason not to expose it.

@IvanGoncharov do you know if there is a good reason for this to be internal only?

From what I can tell it receives a fair bit of use from the community, and it [seems a bit too complex](https://github.com/graphql/graphql-js/blob/47bd8c8897c72d3efc17ecb1599a95cee6bac5e8/src/execution/values.ts#L150-L234) to just copy and paste into a project.

We're currently facing issues with there being no TypeScript declaration for this export, which is preventing us adopting GraphQL v16:

```
src/index.ts:20:10 - error TS2305: Module '"graphql"' has no exported member 'getArgumentValues'.

20 import { getArgumentValues } from "graphql/execution/values";
            ~~~~~~~~~~~~~~~~~


Found 1 error.
```

Surfacing this as a top level export would be appreciated :+1:

--- a/src/execution/index.ts
+++ b/src/execution/index.ts
@@ -15,4 +15,4 @@ export type {
 
 export { subscribe, createSourceEventStream } from './subscribe';
 
-export { getDirectiveValues } from './values';
+export { getVariableValues, getDirectiveValues } from './values';

--- a/src/execution/values.ts
+++ b/src/execution/values.ts
@@ -35,8 +35,6 @@ type CoercedVariableValues =
  * Note: The returned value is a plain Object with a prototype, since it is
  * exposed to user code. Care should be taken to not pull values from the
  * Object prototype.
- *
- * @internal
  */
 export function getVariableValues(
   schema: GraphQLSchema,

--- a/src/index.ts
+++ b/src/index.ts
@@ -318,6 +318,7 @@ export {
   defaultFieldResolver,
   defaultTypeResolver,
   responsePathAsArray,
+  getVariableValues,
   getDirectiveValues,
   subscribe,
   createSourceEventStream,

